cli/permissions.rs

```diff
@@ -735,4 +802,42 @@ mod tests {
     assert_eq!(perms1.request_hrtime(), PermissionState::Deny);
     drop(guard);
   }
+
+  #[test]
+  fn test_fork() {
+    let guard = PERMISSION_PROMPT_GUARD.lock().unwrap();
+    let perms0 = Permissions::from_flags(&Flags {
+      ..Default::default()
+    });
+    set_prompt_result(true);
```

Comment:
Add another test case in the same function that uses `set_prompt_result(false);`